doc/attr.xml

```diff
@@ -902,6 +902,37 @@ gap> DigraphShortestDistances(D);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="UnweightedBellmanFord">
+<ManSection>
+  <Attr Name="UnweightedBellmanFord" Arg="digraph","source"/>
+  <Returns>A list of integers or <K>fail</K>.</Returns>
+  <Description>
+    If <A>digraph</A> is a digraph with <M>n</M> vertices, then this
+    function returns a list with two sublists of <M>n</M> entries, where each entry is
+    either a non-negative integer, or <K>fail</K>.  <P/>
+
+    If there is a directed path from <A>source</A> to vertex <C>i</C>, then for each i-th entry the first sublist contains 
+    the length of the shortest directed path to that i-th vertex and second sublist contains the vertex preceding that i-th 
```

Comment:
This could be clearer - maybe something like "... vertex `<C>i</C>`, then `<C>distances[i]</C>` contains the length of the shortest path from `<C>source</C>` to vertex <C>i</C>, and `<C>predecessors[i]</C>` contains the vertex before `<C>i</C>` on such a shortest path."